plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -203,6 +203,12 @@ public interface CoreProperties {
   String GLOBAL_EXCLUSIONS_PROPERTY = "sonar.global.exclusions";
   String GLOBAL_TEST_EXCLUSIONS_PROPERTY = "sonar.global.test.exclusions";
 
+  /**
+   * @since 9.14
+   */
+  String PROJECT_TESTS_INCLUSIONS_PROPERTY = "sonar.tests.inclusions";
+  String PROJECT_TESTS_EXCLUSIONS_PROPERTY = "sonar.tests.exclusions";
```

Comment:
Ideally we would have the `@since` annotation for both properties.

Reply:
Done